### DISCUSSION TITLE
Developed a free platform to quickly create jsonl datasets for gpt finetuning and customize llm call functions

### DISCUSSION THREAD

**Discussion by Guido1Alessandro1Trevisan** [community/community#130260] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

While I was working on some other projects I created for myself a platform to quickly create jsonl datasets for gpt finetuning and customize llm call functions.  I realized it's quite useful so I might as well just publish the site just in case it could be useful to any of you guys. All the functionalities are client side so you can check easily that I am not trying to steal your datasets :- ) 
Of course completely free!
https://finetune-gpt.vercel.app/

**Comment by ebndev** [comment]
Hi @Guido1Alessandro1Trevisan, thanks for sharing your project with the community!

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
